gpkit/tests/from_paths.py

```diff
@@ -0,0 +1,68 @@
+"Runs each file listed in pwd/TESTS as a test"
```

Comment:
"For more information, see `gpkit/TESTING_README.md`" -- or whatever we decide to call it.